lib/osf-components/addon/components/registries/schema-block-group-renderer/component.ts

```diff
@@ -17,8 +16,8 @@ import template from './template';
 export default class SchemaBlockGroupRenderer extends Component {
     // Required parameters
     schemaBlockGroup!: SchemaBlockGroup;
-    changeset!: Changeset;
     node!: NodeModel;
+    blockStrategy!: Component;
 
     // Optional params
     disabled: boolean = defaultTo(this.disabled, false);
```

Comment:
Add changeset to optional params

Reply:
`changeset` isn't listed as an optional param here?

Reply:
Nevermind, I see that `changeset` and `pageResponses` are now handled by the mappers

Reply:
changesets are now being used in the mappers, so this is a leftover comment from an earlier iteration